client/src/components/Weather/style.css

```diff
@@ -0,0 +1,36 @@
+body{
```

Comment:
for consistency, can we use module.scss files instead (so rename to style.module.scss)? [CSS Modules information](https://css-tricks.com/css-modules-part-1-need/) This should be very easy for you to convert. Every CSS selector must start with the class selector. You can remove the *body* styling (the only one that doesn't have a class selector) and add background-color: white; to .container. Then, in index.js, you can change all of the className="example" to: className={styles.example}

Reply:
You will also need to use camelCase because you cannot get the class from the module in index.js if it is kebab-case

Reply:
![屏幕截图(614)](https://user-images.githubusercontent.com/55303960/111741951-318e4980-88ec-11eb-9cc6-e775d7c86b48.png)
![屏幕截图(615)](https://user-images.githubusercontent.com/55303960/111742099-6dc1aa00-88ec-11eb-8752-4c9501ec5740.png)
@Denijar I follow your advice and make the changes as shown in the image, I think I am doing it wrong cause my css feature is gone, can you see what I am doing wrong here?

Reply:
you need to wrap the `styles.description` in curly braces instead of quote marks. e.g. `className={styles.description}` You also need to have `import styles from `./style.module.scss` at the top of the file